index.html

```diff
@@ -208,6 +211,106 @@ <h3>GetUserMediaSemantics enum</h3>
+            <dt><dfn>avoidInefficientCompressedPixelFormat</dfn> of type
+            <code>sequence&lt;{{boolean}}&gt;</code></dt>
+            <dd>
+              <p>If the source has no inefficient compressed pixel formats, a
+              single <code>true</code> is reported. If the source only has
+              inefficient compressed pixel formats, a single <code>false</code>
+              is reported. If the script can control the feature, the source
+              reports a list with both <code>true</code> and <code>false</code>
+              as possible values. See <a href=
+              "#def-constraint-avoidInefficientCompressedPixelFormat">
+              avoidInefficientCompressedPixelFormat</a> for additional
+              details.</p>
+            </dd>
+          </dl>
+        </section>
+      </div>
+    </section>
+    <section id="mediatracksettings-dictionary-extensions">
+      <h3>MediaTrackSettings dictionary extensions</h3>
+      <div>
+        <pre class="idl"
+>partial dictionary MediaTrackSettings {
+  boolean avoidInefficientCompressedPixelFormat;
+};</pre>
+        <section>
+          <h2>Dictionary {{MediaTrackSettings}} Members</h2>
+          <dl data-link-for="MediaTrackSettings" data-dfn-for=
+          "MediaTrackSettings" class="dictionary-members">
+            <dt><dfn>avoidInefficientCompressedPixelFormat</dfn> of type
+            {{boolean}}</dt>
+            <dd>See <a href=
+            "#def-constraint-avoidInefficientCompressedPixelFormat">
+            avoidInefficientCompressedPixelFormat</a> for details.</dd>
+      </section>
+      </div>
+    </section>
+      <h2>Constrainable Properties</h2>
+      <h3>Additions to Constrainable Properties</h3>
+      <p>The names of constrainable properties in this document are defined
+      below.</p>
+      <table class="simple">
+        <thead>
+          <tr>
+            <th>Property Name</th>
+            <th>Values</th>
+            <th>Notes</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td><dfn id="def-constraint-avoidInefficientCompressedPixelFormat">
+            avoidInefficientCompressedPixelFormat</dfn></td>
+            <td>{{ConstrainBoolean}}</td>
+            <td>
+              <p>Compressed pixel formats need to be decoded. The user agent
+              SHOULD label compressed pixel formats that incur significant power
+              penalty when decoded as not power efficient. The labeling is up to
+              the user agent, but decoding MJPEG in software is an example of an
+              expensive mode.</p>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <h3>Additions to allowed required constraints for device selection</h3>
+      <p>The following constrainable properties are
+      <a data-cite="mediacapture-streams">allowed required constraints for device selection</a>:
+      <a>avoidInefficientCompressedPixelFormat</a>.
```

Comment:
1. Do we really want this in "allowed required constraints for device selection" or should I remove this part?
2. I couldn't get the data citing for this reference to work, the best I got was linking to the correct document. I think the problem is that mediacapture-main's definition of this does not have the "data-export" attribute in the &lt;dfn&gt;, if so we need to do a PR to mediacapture-main as well.

Reply:
We do not want this in the "allowed required constraints for device selection" list.

Reply:
Got it. Removed.